app/controllers/pulitzer/posts_controller.rb

```diff
@@ -24,10 +24,14 @@ def show
 
   def edit
     if request.xhr?
-      render partial: 'form', locals: { post: @post }
+      render 'edit', locals: { post: @post }
```

Comment:
I don't think you can pass `locals` without using `render partial: ...`
Has that changed in rails?